### ISSUE TITLE
Script "Mission faction state"

### ISSUE THREAD

**Issue by StivlWrith** [EDCD/EDDI#969]
## EDDI version in which issue found
3.1.1
### Gravity
Very, very low

There is something i don't understand in this script "_Mission faction state_", 
in line 9 there is
```
|elif (mission.factionstate) = "Democracy":
    {OneOf("en the name of","for")} democracy
```
Isn't "_Democracy_" more of a government type than a faction state?

same line 15
|elif (mission.factionstate) = "Industrial":

_Industrial_ is more an economic data

**Comment by Darkcyde13** [comment]
When I first made my Missions Systems (which this is based off), I went through all the faction state types in the game.  Both Democracy and Industrial were in that list, along with things like Planet, Refinery, Skimmer, and Surface.  As odd as it was, I included everything for completeness, but I didn't assign speech to them all.

Of course, this is going back almost two years now, so It's quite possible these have been updated in-game and are no longer needed.  The others I mentioned have been removed by the EDDI team, so it's possible these two were just missed.

**Comment by StivlWrith** [comment]
Yes, many changes in each versions, and sometimes, It's seems sometimes thet they remove or change something to return in old way one or two version later, hard to follow for every body

**Comment by Tkael** [comment]
We changed the code in #890 so that `mission.factionstate` will only output valid states. 
Version 3.1.1 will never see `(mission.factionstate) = "Democracy"` as true.
@Hoodathunk, have you got this? It looks like the Cottle script may need a bit of a polish.

**Comment by Hoodathunk** [comment]
Resolved by PR #1300, Commit #d4a6b54.

**Comment by Tkael** [comment]
Merged.
